templates/imgproxy/index.ts

```diff
@@ -0,0 +1,71 @@
+import { createTemplate, Services, randomString } from "~templates-utils";
+
+export default createTemplate({
+  name: "imgproxy",
+  meta: {
+    description: "imgproxy is the blazing fast and secure image processing tool.",
```

Comment:
Let's make the description longer. Several ideas from their website

- imgproxy resizes and processes images on the fly and doesn’t consume disk space
- imgproxy uses the world’s fastest image processing library under the hood — libvips. It is screaming fast and has a tiny memory footprint.